packages/cli/src/tools/config/__tests__/index-test.ts

```diff
@@ -79,8 +105,7 @@ test('should return dependencies from package.json', () => {
 
 test('should read a config of a dependency and use it to load other settings', () => {
   writeFiles(DIR, {
-    'node_modules/react-native/package.json': '{}',
-    'node_modules/react-native-test/package.json': '{}',
```

Comment:
why this is removed?

Reply:
Accident, but I am surprised that the tests are passing.